### ISSUE TITLE
Bug - TextInput - aria-expanded not set

### ISSUE THREAD

**Issue by phill-85** [patternfly/patternfly-react#9305]
**Describe the problem**

The [Accessibility](https://pf5.patternfly.org/components/forms/form-control#accessibility) table for the `TextInput` component suggests that the `aria-expanded` attribute should be set, however when using the flag `isExpanded` it is not.
